examples/stm32/WolfSSL/.cproject

```diff
@@ -3,6 +3,9 @@
 	<storageModule moduleId="org.eclipse.cdt.core.settings">
```

Comment:
we don't need these changes, I just implemented the relative path solution which totally eliminates edit needs by the Tutorial reader, pls revert this file, thx